dashboard/modules/job/common.py

```diff
@@ -95,6 +95,8 @@ class JobInfo:
     #: The node id that driver running on. It will be None only when the job status
     # is PENDING, and this field will not be deleted or modified even if the driver dies
     driver_node_id: Optional[str] = None
+    #: The driver process exit code after driver terminates
```

Comment:
Please define in the comment when this would be None.  I think this is what shows up in the auto-generated Ray docs, but it would be good to confirm this and see if there's anywhere else in the docs that we need to update